mmengine/registry/registry.py

```diff
@@ -362,24 +414,6 @@ def get(self, key: str) -> Optional[Type]:
                         break
                     parent = parent.parent
         else:
-            try:
-                module = import_module(f'{scope}.utils')
-                module.register_all_modules(False)  # type: ignore
-            except (ImportError, AttributeError, ModuleNotFoundError):
-                if scope in PKG2PROJECT:
-                    print_log(
-                        f'{scope} is not installed and its modules '
-                        'will not be registered. If you want to use '
-                        f'modules defined in {scope}, Please install '
-                        f'{scope} by `pip install {PKG2PROJECT[scope]} ',
-                        logger='current',
-                        level=logging.WARNING)
-                else:
-                    print_log(
-                        f'Failed to import "{scope}", and register its '
-                        f'modules. Please register {real_key} manually.',
-                        logger='current',
-                        level=logging.WARNING)
```

Comment:
Could we move https://github.com/open-mmlab/mmengine/pull/643/files#diff-df9a9f722ab32299254fad7406178ad6e0a9e71dbc689d11af151999542f1ec2R384-R398 to here?

Reply:
Yep!